Buildings/Controls/OBC/CDL/Continuous/Validation/AMIGOWithFOTD.mo

```diff
@@ -0,0 +1,66 @@
+within Buildings.Controls.OBC.CDL.Continuous.Validation;
+model AMIGOWithFOTD "Validation model for the AMIGOWithFOTD block"
+
+  Buildings.Controls.OBC.CDL.Continuous.AMIGOWithFOTD FOTD(controllerType=Buildings.Controls.OBC.CDL.Types.SimpleController.PI)
+    annotation (Placement(transformation(extent={{20,-10},{40,10}})));
+  Modelica.Blocks.Sources.Sine sine(
+    amplitude=1.2,
+    f=1/60,
+    offset=0.3,
+    startTime=0.025)                                      "Sine source"
+    annotation (Placement(transformation(extent={{-82,30},{-62,50}})));
+  Modelica.Blocks.Sources.Constant const(k=0) annotation (Placement(transformation(extent={{-82,-10},{-62,10}})));
+  Buildings.Controls.OBC.CDL.Continuous.NormalizedDelay NormalizedDelay(gamma=3) annotation (Placement(transformation(extent={{-40,-60},{-20,-40}})));
+  Discrete.Relay                            relay(
+    yHigher=4,
+    yLower=-0.5,
+    deadBand=0.5)
+                annotation (Placement(transformation(extent={{-40,20},{-20,40}})));
```

Comment:
add missing comment

Reply:
Comments are added to all the blocks of the models in this package